apps/homescreen/js/app.js

```diff
@@ -540,6 +577,12 @@ const SETTINGS_VERSION = 0;
 
         this.pageHeight = pageHeight;
         this.pendingGridHeight = gridHeight;
+
+        if (!this.visualLoadComplete &&
+            Math.floor(visibleChildren / (this.small ? 4 : 3)) * iconHeight >=
+              scrollHeight) {
+          this.onVisualLoad();
+        }
```

Comment:
That is weird... I don't see that, but I'm also not using the software home button. I say let's fix this in a follow-up, I'm pretty sure the maths checks out so this could be a slightly different problem...